src/region/RegionCreator.tsx

```diff
@@ -218,12 +206,24 @@ export default function RegionCreator({ canDraw, className, onStart, onStop }: P
     return (
         <div
             ref={creatorElRef}
-            className={classNames(className, 'ba-RegionCreator', { 'is-active': canDraw })}
+            className={classNames(className, 'ba-RegionCreator')}
             data-testid="ba-RegionCreator"
-            {...eventHandlers}
+            onBlur={noop}
+            onClick={handleClick}
+            onFocus={noop}
+            onKeyPress={noop}
+            onMouseDown={handleMouseDown}
+            onMouseOut={handleMouseOut}
+            onMouseOver={handleMouseOver}
+            onTouchCancel={handleTouchCancel}
+            onTouchEnd={handleTouchEnd}
+            onTouchMove={handleTouchMove}
+            onTouchStart={handleTouchStart}
+            role="button"
+            tabIndex={-1}
```

Comment:
We should avoid tabindex on this element, as it isn't meant to be interacted with via the keyboard.